cmd/podman/top.go

```diff
@@ -18,7 +18,9 @@ var (
 	topDescription = `
    podman top
```

Comment:
Indentation is wierd here - looks like this is using spaces while the new description uses tabs?

Reply:
Good catch! I will fix that and make it consistent with other commands (e.g., `podman-pull`).